src/routes/alerts/alerts.controller.spec.ts

```diff
@@ -112,6 +117,8 @@ describe('Alerts (Unit)', () => {
       const moduleFixture: TestingModule = await Test.createTestingModule({
         imports: [AppModule.register(testConfiguration)],
       })
+        .overrideModule(JWT_CONFIGURATION_MODULE)
```

Comment:
As this opts in to `features.email`, the override is needed as`AuthGuard` is now on the `EmailController`.